01-js-intro-and-jquery/05/css/styles.css

```diff
@@ -0,0 +1,7 @@
+.hidden {
+	display: none;
+}
+
+section {
```

Comment:
always try to use class selectors instead of tag selectos. Not need to change for this exercise, just be aware for next exercises.